core/src/main/java/org/wso2/testgrid/core/command/GenerateTestPlanCommand.java

```diff
@@ -557,7 +558,8 @@ private String createTestPlanGenDirectory(Product product) throws CommandExecuti
         try {
             String directoryName = product.getName();
             String testGridHome = TestGridUtil.getTestGridHomePath();
-            Path directory = Paths.get(testGridHome, directoryName, PRODUCT_TEST_PLANS_DIR).toAbsolutePath();
+            Path directory = Paths.
+                    get(testGridHome, TESTGRID_JOB_DIR, directoryName, PRODUCT_TEST_PLANS_DIR).toAbsolutePath();
```

Comment:
Shall we introduce a TestgridUtil method for this?

Reply:
+1 Will include for the next relevant PR. :)